frontend/src/components/admin/AssignedStoryTranslationsTable.tsx

```diff
@@ -0,0 +1,176 @@
+
+        newStoryTranslations.sort(roleSort);
+        break;
+      }
+      case "language": {
+        setIsAscendingLanguage(!isAscendingLanguage);
+
+        const languageSort = (t1: StoryTranslation, t2: StoryTranslation) =>
+          isAscendingLanguage
+            ? t1.language.localeCompare(t2.language)
+            : t2.language.localeCompare(t1.language);
+
+        newStoryTranslations.sort(languageSort);
+        break;
+      }
+      case "stage": {
+        setIsAscendingStage(!isAscendingStage);
+
+        const stageSort = (t1: StoryTranslation, t2: StoryTranslation) =>
+          isAscendingStage
+            ? t1.stage.localeCompare(t2.stage)
+            : t2.stage.localeCompare(t1.stage);
+
+        newStoryTranslations.sort(stageSort);
+        break;
+      }
+      default: {
+        break;
+      }
+    }
+    setStoryTranslations(newStoryTranslations);
+  };
+
+  const tableBody = storyTranslations.map(
+    (translation: StoryTranslation, index: number) => (
+      <Tr
+        borderBottom={
+          index === storyTranslations.length - 1 ? "1em solid transparent" : ""
+        }
+        key={`${translation?.storyTranslationId}`}
+      >
+        <Td>{translation.title}</Td>
+        <Td>{getRole(translation)}</Td>
+        <Td>
+          <Badge
+            background={getLevelVariant(translation.level)}
+            marginBottom="3px"
+            marginTop="3px"
+          >{`${convertLanguageTitleCase(translation.language)} | Level ${
+            translation.level
+          }`}</Badge>
+        </Td>
+        <Td>{convertStageTitleCase(translation.stage)}</Td>
+        <Td>
+          <IconButton
+            aria-label={`Delete story translation for ${translation.title}`}
+            background="transparent"
+            icon={<Icon as={MdDelete} />}
+            width="fit-content"
+          />
+        </Td>
+      </Tr>
+    ),
+  );
+  return (
+    <Table
+      borderRadius="12px"
+      boxShadow="0px 0px 2px grey"
+      size="sm"
+      theme="gray"
+      variant="striped"
+      width="100%"
+    >
+      <Thead>
+        <Tr
+          borderTop="1em solid transparent"
+          borderBottom="0.5em solid transparent"
+        >
+          <Th cursor="pointer" onClick={() => sort("title")}>
+            {`BOOK TITLE ${isAscendingTitle ? "↑" : "↓"}`}{" "}
+          </Th>
+          <Th cursor="pointer" onClick={() => sort("role")}>{`ROLE ${
+            isAscendingRole ? "↑" : "↓"
+          }`}</Th>
+          <Th
+            cursor="pointer"
+            onClick={() => sort("language")}
+          >{`LANGUAGE & LEVEL ${isAscendingLanguage ? "↑" : "↓"}`}</Th>
+          <Th cursor="pointer" onClick={() => sort("stage")}>{`PROGRESS ${
+            isAscendingStage ? "↑" : "↓"
+          }`}</Th>
+          <Th>ACTION</Th>
+        </Tr>
+      </Thead>
+      <Tbody>{tableBody}</Tbody>
+      {/* <hr marginTop="-10px"/>  How do I add a horizontal line lol */}
```

Comment:
the figma has a light grey horizontal line here, not too sure how to properly implement this

Reply:
I wouldnt worry about the horizontal line - imo the alternating cell shading makes it quite clear that the button is unrelated to any given row

Reply:
lets see what the designers say tho